app/views/assets/_asset.html.erb

```diff
@@ -1,8 +1,29 @@
-<% gallery_view_id = nil unless defined? gallery_view_id %>
+<%
+gallery_view_id = nil unless defined? gallery_view_id
+
+unless defined? deletable
+  deletable = if asset.step.present?
```

Comment:
Sorry, I meant `deletable = asset.step.present?` instead of the whole "if" block.